src/NuProj.Tasks/AssignTargetFramework.cs

```diff
@@ -37,7 +37,7 @@ where seenPackagePaths.Add(packagePath)
             return true;
         }
 
-        private static ITaskItem ConvertToPackageFile(ITaskItem output)
+        private static ITaskItem ConvertToPackageFile(ITaskItem output, Task task)
```

Comment:
Would prefer to change the method to not-static, instead of passing `this` as parameter.